odoo/addons/base/models/ir_actions.py

```diff
@@ -684,6 +705,58 @@ def _compute_warning(self):
             else:
                 action.warning = False
 
+    @api.model
+    def _get_children_domain(self):
+        domain = Domain([
+            ("model_id", "=", unquote("model_id")),
```

Comment:
I'm not familiar with the new Domain classes, but this has little chance to work.
For dynamic stuff I think strings are still necessary

right @kmagusiak ?

Moreover, unquote is in the translate.py module and doesn't seem to apply here.